aiChatterBox/aiChatterBox/MainWindow.xaml.cs

```diff
@@ -0,0 +1,284 @@
+﻿//Sauraav Jayrajh
+//sauraavjayrajh@gmail.com
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace aiChatterBox
+{
+    public partial class MainWindow : Window
+    {
+        //Client to Handle API Interactions
+        private static readonly HttpClient client = new HttpClient();
+        //Adress of Ollama Running On Machine, Change if Needed
+        public static string localhost = "http://localhost:11434";
```

Comment:
Consider having a way to pull from a configuration file, then have an control or form that allows the user to update this value.